### PR TITLE
ci: change for run trigger

### PR DESCRIPTION
**Pull request by hamidrezaramzani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
-.
+..
```
Comment (navidnsry) [inline README.md:2]: test

**Review by navidnsry — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 ## Getting Started
 
 First, run the development server:
```
Comment (testhamidreza) [inline README.md:5]: test

**Review by testhamidreza — CHANGES_REQUESTED** [review]

**Review by testhamidreza — APPROVED** [review]

File: .github/workflows/pr-notifier.yml [hunk 3]
```diff
@@ -11,43 +11,50 @@ jobs:
...
+        env:
+          DISCORD_WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
+        run: |
+          MESSAGE="$GITHUB_EVENT_PULL_REQUEST_TITLE
+          > :bust_in_silhouette: **User**: ${{ github.event.pull_request.assignee.login }}
+          :herb: **Branch**: ${{ github.event.pull_request.head.ref }}
+          :deciduous_tree: **Base Branch**: ${{ github.event.pull_request.base.ref }}
+          :open_file_folder: **Repository**: ${{ github.repository }}
+          :link: **Pull Request Link**: ${{ github.event.pull_request.html_url }}
+          :clipboard: **Activity**: Assigned"
```
Comment (testhamidreza) [inline .github/workflows/pr-notifier.yml]: test

**Review by testhamidreza — CHANGES_REQUESTED** [review]

**Review by testhamidreza — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
-.
+..
 ## Getting Started
```
Comment (testhamidreza) [inline README.md:4]: test

**Review by testhamidreza — APPROVED** [review]

**Review by testhamidreza — APPROVED** [review]

**Review by testhamidreza — APPROVED** [review]

**Review by testhamidreza — APPROVED** [review]

**Review by testhamidreza — APPROVED** [review]

**Review by testhamidreza — APPROVED** [review]

**Review by testhamidreza — APPROVED** [review]

**Review by testhamidreza — APPROVED** [review]

**Pull request closed without merging**